src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -100,10 +100,16 @@ def __init__(self, config):
 
         self.packageDir = None
 
-        if os.path.exists(os.path.join(getWMBASE(),
-                                       'src/python/WMCore/WMRuntime/Unpacker.py')):
-            self.unpacker = os.path.join(getWMBASE(),
-                                         'src/python/WMCore/WMRuntime/Unpacker.py')
+        # if agent is running in a container, Unpacker.py must come from a directory
+        # on the host so the condor schedd can see it
+        # config.General.workDir should always be bind mounted to the container
+        if config.Agent.isDocker:
```

Comment:
One more comment here, how about changing it to:
`if getattr(config.Agent, "isDocker", False):`
?
With such change, you don't necessarily need to update those test files/configs

Reply:
Would you prefer I leave out the updates to the unit test files then?

Reply:
Given that they are already in place, it doesn't harm to leave them now.